.pipelines/build.yml

```diff
@@ -0,0 +1,40 @@
+# Azure DevOps Pipeline running build
+variables:
+  - template: vars.yml
+
+jobs:
+  - job: Build_Push_Images
+    pool:
+      vmImage: "ubuntu-latest"
+    strategy:
+      matrix:
+        RP:
+          makefile.target: "image-aro"
+        Proxy:
+          makefile.target: "image-proxy"
+        MDM:
+          makefile.target: "image-mdm"
+
+    steps:
+      - template: ./templates/template-setup-golang-env.yml
+        parameters:
+          gobin: ${{ variables.GOBIN }}
+          gopath: ${{ variables.GOPATH }}
+          goroot: ${{ variables.GOROOT }}
+          modulePath: ${{ variables.modulePath }}
+      - script: |
+          # az login -u $(ARO_CI_SPN_ID) -p $(ARO_CI_SPN_PASSWORD) --tenant $(ARO_CI_SPN_TENANT) --service-principal
+          # az account set -s $(ARO_ACR_SUB_ID)
+          # --> registry.access.redhat.com and arosvc.azurecr.io <--
+          # az keyvault secret show --id $(ARO_ACR_CI_SECRET) | jq -r .value > ~/.docker/config.json
+          echo az login + acr pull secret from KV
```

Comment:
For now just doing echo command, just want to validate the logic here.
We also need a way to pull ` registry.access.redhat.com and arosvc.azurecr.io` secrets. Was thinking to use a dedicated KV for CI / E2E WDYT ?

Reply:
I also need a way to store secrets for the e2e tests.

Reply:
@asalkeld good to know, new sub are on the way ! 
As soon we get it i'll let you know.